articles/history-of-nodejs/history-of-nodejs.md

```diff
@@ -0,0 +1,167 @@
+---
+layout: engineering-education
+status: publish
+published: true
+url: /engineering-education/history-of-nodejs/
+title: The History of Node.js
+description: Node.js is a runtime server environment that uses JavaScript on the server side and asynchronous programming. It is a free and open source technology that runs on various platforms (Mac OS X, Unix, Windows, etc.)
+author: jethro-magaji
+date: 2020-08-20T00:00:00-08:00
+topics: [Node.js]
+excerpt_separator: <!--more-->
+images:
+
+  - url: /engineering-education/history-of-nodejs/hero.jpg
+    alt:
+---
+Today we are going to talk about a revolutionary technology that is being used to build great web applications, it is called  *Node.js*.
+You may have probably heard of it or worked with it (or have been curious about it in the past), either way you are welcome to join us as we further explore the History of Node.js in this article. We are going to talk about the history, some of the basics, and a few use case examples that use this revolutionary technology.
+<!--more-->
+![Nodejs image](/engineering-education/history-of-nodejs/nodejs-use-cases-cover-image.png)
+[source](https://www.simform.com/nodejs-use-case/)
+
+### What is Node.js?
+![Nodejs logo](https://drive.google.com/uc?export=view&id=13udtjL4Xz5YICREzII6hcHiv0xM8a_ds)
```

Comment:
please include all images within the article itself - within the file folder - see image above as an example